R/setkey.R

```diff
@@ -310,6 +310,7 @@ setorder <- function(x, ..., na.last=FALSE)
 
 setorderv <- function(x, cols, order=1L, na.last=FALSE)
 {
+  if (missing(cols)) cols = colnames(x)
```

Comment:
As discussed in #3169; not setting as default argument for now as that may be a breaking change.

Reply:
what breaking change it would be precisely?

Reply:
I dunno. I guess _adding_ a default argument is different from _removing_ a default argument. Will fix